packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -430,6 +430,43 @@ void main() {
     expect(MediaQuery.of(popupContext).padding, EdgeInsets.zero);
   });
 
+  testWidgets('Popup Menu Offset Test', (WidgetTester tester) async {
+    const Offset offset = Offset(0.0, 40.0);
+
+    final PopupMenuButton<int> popupMenuButton =
+      new PopupMenuButton<int>(
+        offset: offset,
+        itemBuilder: (BuildContext context) {
+          return <PopupMenuItem<int>>[
+            new PopupMenuItem<int>(
+              value: 1,
+              child: new Builder(
+                builder: (BuildContext context) {
+                  return const Text('AAA');
+                },
+              ),
+            ),
+          ];
+        },
+      );
+
+    await tester.pumpWidget(new MaterialApp(
+        home: new MediaQuery(
+          data: const MediaQueryData(
+            padding: EdgeInsets.all(50.0),
+          ),
+          child: new Material(
+            child: popupMenuButton,
+          ),
+        )
+    ));
+
+    await tester.tap(find.byType(IconButton));
+    await tester.pumpAndSettle();
+    
+    expect(tester.getTopLeft(find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_PopupMenu<int>')), const Offset(8.0, 40.0));
```

Comment:
Do we know what the 8 is from? Would add a comment if possible.

Reply:
I'm not sure... What it looks to me actually, is for this specific popup button we can actually only modify if it is positioned on the left or right side of the screen, and then it's vertical position on the screen.